referenceframe/model_test.go

```diff
@@ -38,35 +38,6 @@ func TestModelLoading(t *testing.T) {
 	test.That(t, m.Name(), test.ShouldEqual, "foo")
 }
 
-func TestTransform(t *testing.T) {
```

Comment:
why do we not need this test anymore?

Reply:
kind of just a useless test tbh. Everything tested here is done elsewhere too and its not even testing the function properly